airflow/cli/commands/connection_command.py

```diff
@@ -177,16 +185,19 @@ def connections_add(args):
     missing_args = []
     invalid_args = []
     if args.conn_uri:
+        if not _valid_uri(args.conn_uri):
+            msg = f'The URI provided to --conn-uri is invalid: {args.conn_uri}'
+            raise SystemExit(msg)
         for arg in alternative_conn_specs:
             if getattr(args, arg) is not None:
                 invalid_args.append(arg)
     elif not args.conn_type:
         missing_args.append('conn-uri or conn-type')
     if missing_args:
-        msg = 'The following args are required to add a connection:' + f' {missing_args!r}'
+        msg = f'The following args are required to add a connection: {missing_args!r}'
```

Comment:
This line and line 200 are not really related to this PR's scope. But fixing them is too minor and maybe not worth creating separate PR for them. So fixing them together here.